src/runtime_src/core/pcie/driver/linux/xocl/devices.h

```diff
@@ -138,33 +140,43 @@ enum {
 
 #endif
 
-#define	MGMT_SUFFIX		".m"
-#define	USER_SUFFIX		".u"
+#define	MGMTPF		0
+#define	USERPF		1
+
+#if PF == MGMTPF
+#define SUBDEV_SUFFIX	".m"
+#elif PF == USERPF
+#define SUBDEV_SUFFIX	".u"
+#endif
 
 #define XOCL_FEATURE_ROM	"rom"
-#define XOCL_XDMA			"dma.xdma"
-#define XOCL_QDMA			"dma.qdma"
+#define XOCL_IORES0		"iores0"
```

Comment:
we have only three iores subdevices?

Reply:
each of them contains multiple IPs. iores0 is for IPs defined in devices.h. iores1 is for BLP and iores2 is for PRP